sql/core/src/test/resources/sql-tests/results/identifier-clause.sql.out

```diff
@@ -535,7 +535,7 @@ Catalog Name	spark_catalog
 Comment	some comment
 Location [not included in comparison]/{warehouse_dir}/someloc
 Namespace Name	ident
-Owner	runner
+Owner	maximgekk
```

Comment:
Probably it is related to https://github.com/apache/spark/pull/41007. @srielau @gengliangwang Is it possible to rewrite the test in a way to don't depend of local settings. Otherwise it will bother every time while re-generating golden files.